attributes/default.rb

```diff
@@ -4,6 +4,7 @@
 default.loggly.tags = []
 
 default.loggly.log_files = []
+default.loggly.apps = {}
```

Comment:
The "app" concept is meant to construct files similar to the example from here:
https://www.loggly.com/docs/nginx-server-logs/
under the section "2. Setup Nginx File Monitoring"

Basically, it allows the grouping of multiple files within the same "tag".  There are similar examples in the docs, for example in the apache section.  

There's no loss of granularity because events from each individual file are still tagged with the %app_name% (in the example from the loggly docs, nginx-access and nginx-error.

If you can think of a better name than "apps", I'm happy to hear it.  Could call it something like file_groups or file_sets to be more explicit.